source/components/index.markdown

```diff
@@ -88,8 +88,19 @@ Entities are things that you want to observe within Home Assistant. Support for
   <td>Track what is being played and control playback. (temporarely disabled awaiting protobuf 3 release)</td>
 </tr>
 
+<tr>
```

Comment:
Yeah, I was back and forth on that myself. I was worried that under the Hubs category, the UDI may not be enough to alert someone to the Insteon and X10 compatibility. I'll try putting Insteon and X10 in the entities section and the ISY994 under hubs. We can see how that looks.